frontend/src/scenes/funnels/funnelLogic.ts

```diff
@@ -503,7 +500,6 @@ export const funnelLogic = kea<funnelLogicType>({
                     ...cleanedParams,
                     funnel_window_interval: conversionWindow.funnel_window_interval,
                     funnel_window_interval_unit: conversionWindow.funnel_window_interval_unit,
-                    ...(!featureFlags[FEATURE_FLAGS.FUNNEL_BAR_VIZ] ? { breakdown: null, breakdown_type: null } : {}),
```

Comment:
I don't think breakdown is available yet on time conversion or trends. Will passing non-empty breakdown parameters to the API break anything for these graphs?

Reply:
We've encountered some weirdness with this being set in requests to the backend before, which _did_ break the response, but should be alright now as that prompted us to add bits to ignore breakdown attributes for trends/time-to-convert (until breakdown is actually supported).

Reply:
Definitely we should keep the request clean on the frontend too even if the backend ignores it, but I removed this line because the condition is always false, therefore we end up spreading just an empty object.